### PR TITLE
qemu.tests: Improve migration_multi_host

### PR DESCRIPTION
**Pull request by spcui**
1. Add check guest doesn't reboot during multi-host migration
2. Clear unused imports

### CODE REVIEW

File: qemu/tests/migration_multi_host.py [hunk 1]
```diff
@@ -48,6 +55,17 @@ def run(test, params, env):
...
+        logging.info("Try to login guest before migration test...")
+        vm = env.get_vm(vms[0])
+        if vm is None:
+            env_process.preprocess_vm(test, params, env, vms[0])
+            vm = env.get_vm(vms[0])
+
+        session = vm.wait_for_login(timeout=login_timeout)
+        logging.info("Start the background command in the guest...")
+        session.sendline(migration_bg_command)
+        time.sleep(5)
```
Comment (ypu) [inline qemu/tests/migration_multi_host.py]: Another note: We have some tests that don't want to login into guest at the first stage. So please make this configurable when you add the code.

File: qemu/tests/migration_multi_host.py [hunk 2]
```diff
@@ -34,31 +28,83 @@ def run(test, params, env):
...
+            self.mig_bg_kill_command = params.get("migration_bg_kill_command",
+                                                  "pkill -9 ping")
+            self.need_to_login = params.get("need_to_login", "no")
+
+        def run_pre_sub_test(self):
+            # is source host
+            if self.is_src:
+                if self.pre_sub_test:
+                    if self.login_before_pre_tests == "yes":
+                        vm = env.get_vm(params["main_vm"]) vm.wait_for_login(timeout=self.login_timeout)
```
Comment (xutian) [inline qemu/tests/migration_multi_host.py]: it looks a syntax issue

**Comment by xutian** [comment]
LGTM

Acked-by: @xutian

**Comment by FengYang** [comment]
Thanks @xutian  and @spcui , will merge it.

**Pull request merged**